src/main/mdo/assembly.mdo

```diff
@@ -226,7 +226,7 @@
         </field>
         <field>
           <name>repositories</name>
-          <version>1.0.0+</version>
```

Comment:
This was 1.0.0 and unintentional change (copy+paste) happened, this should be `1.0.0/2.1.1`.